client/app/components/authors/HeaderContainer.jsx

```diff
@@ -45,19 +46,21 @@ const HeaderContainer = ({
                         </div>
                     )}
                 </div>
-                <div className="right">
-                    <button
-                        className="button button--menu-icon"
-                        aria-label="Menu"
-                        aria-controls="navigation"
-                        type="button"
-                    >
-                        {isMobileMenuOpen
-                            ? <SVG src={IcClose} onClick={() => setIsMobileMenuOpen(false)} />
-                            : <SVG src={IcMenu} onClick={() => setIsMobileMenuOpen(true)} />}
-                    </button>
-                    {renderMenu(true)}
-                </div>
+                { (!author || shouldShowMenu(author, pages)) && (
+                    <div className="right">
+                        <button
+                            className="button button--menu-icon"
+                            aria-label="Menu"
+                            aria-controls="navigation"
+                            type="button"
+                        >
+                            {isMobileMenuOpen
+                                ? <SVG src={IcClose} onClick={() => setIsMobileMenuOpen(false)} />
+                                : <SVG src={IcMenu} onClick={() => setIsMobileMenuOpen(true)} />}
+                        </button>
+                        {renderMenu(true)}
+                    </div>
+                )}
             </div>
             {renderMenu(false)}
```

Comment:
Should we also add the check for shouldShowMenu here? Otherwise the desktop menu will always be displayed

Reply:
mm yeah, it makes sense!

Reply:
fixed! @antsgar